dabest/_classes.py

```diff
@@ -1325,6 +1350,7 @@ def __init__(self, dabest, effect_size,
         self.__dabest_obj        = dabest
         self.__effect_size       = effect_size
         self.__is_paired         = is_paired
+        self.__is_preceding      = is_preceding
```

Comment:
Please change `preceding` to `repeated_measures` and `is_preceding` to ` repeated_measures`